val/src/avs_wd.c

```diff
@@ -43,6 +43,10 @@ val_wd_execute_tests(uint32_t level, uint32_t num_pe)
       }
   }
 
+  if (g_single_module != SINGLE_MODULE_SENTINEL && g_single_module != AVS_WD_TEST_NUM_BASE) {
+    return AVS_STATUS_SKIP;
```

Comment:
Add log print for consistency in log
val_print(AVS_PRINT_TEST, "      USER Override - Skipping all Watchdog tests (running only a single module)\n", 0);

Reply:
Fixed.